setup.py

```diff
@@ -20,7 +20,7 @@
 
 setup(
     name='django-essentials',
-    version='1.5.0',
+    version='1.5.1',
```

Comment:
We could jump to version 1.6 since we are targeting an all new Django LTS version.